app/models/organisation.rb

```diff
@@ -3,7 +3,10 @@ class Organisation < ApplicationRecord
   has_many :funds
 
   validates_presence_of :name, :organisation_type, :language_code, :default_currency
-  validates :iati_reference, uniqueness: {case_sensitive: false}, presence: true
+  validates :iati_reference,
+    uniqueness: {case_sensitive: false},
+    presence: true,
+    format: {with: /\A[a-zA-Z]{2,}-[a-zA-Z]{3}-.+\z/, message: "Identifiers must start with a country code and company type separated by a dash, eg. GB-GOV-13"}
```

Comment:
You're gonna hate me but..... should this be internationalised?

Reply:
Ha, yeah it should be! Thanks for calling me out. I'm on it 🏃